cfgov/scripts/initial_data.py

```diff
@@ -4,18 +4,72 @@
 
 from django.contrib.auth.hashers import make_password
 from django.contrib.auth.models import User
+from django.db import transaction
+from wagtail.wagtailcore.models import Page, Site
 
+from v1.models import HomePage
 
+
+@transaction.atomic
 def run():
-    print('Running script \'scripts.initial_data\' ...')
+    print('Running script initial_data')
```

Comment:
I know your changeset didn't introduce this, but would this be a good time to update the print statements to log statements?  An added benefit would be that we'd have a way of checking if it was accidentally run in prod.